tests/functional/NotificationTemplate.php

```diff
@@ -72,4 +73,57 @@ public function testClone()
 
         $this->array($template->fields)->isIdenticalTo($clonedTemplate->fields);
     }
+
+    protected function getTextFromConverter(): iterable
+    {
+        $base_url = GLPI_URI;
+
+        yield [
+            'content' => <<<HTML
+Relative link from GLPI: <a href="/front/comuter.php?id=2">Computer</a>
+HTML,
+            'expected' => <<<HTML
+Relative link from GLPI: <a href="{$base_url}/front/comuter.php?id=2">Computer</a>
+HTML,
+        ];
+
+        yield [
+            'content' => <<<HTML
+Absolute link from GLPI: <a href="{$base_url}/front/comuter.php?id=2">Computer</a>
+HTML,
+            'expected' => <<<HTML
+Absolute link from GLPI: <a href="{$base_url}/front/comuter.php?id=2">Computer</a>
+HTML,
+        ];
+
+        yield [
+            'content' => <<<HTML
+External link from GLPI: <a href="https://faq.teclib.com/01_getting_started/getting_started/">Faq</a>
+HTML,
+            'expected' => <<<HTML
+External link from GLPI: <a href="https://faq.teclib.com/01_getting_started/getting_started/">Faq</a>
+HTML,
+        ];
+
+        yield [
+            'content' => <<<HTML
+External link without protocol from GLPI: <a href="faq.teclib.com/01_getting_started/getting_started/">Faq</a>
```

Comment:
Links relative to current scheme are starting with a `//`.
```suggestion
External link without protocol from GLPI: <a href="//faq.teclib.com/01_getting_started/getting_started/">Faq</a>
```